docsrc/admin/installation.md

```diff
@@ -90,45 +88,35 @@ If you feel SELinux is important to the security of your server, please start a
 
         cp ansible_collections/iridl/iridl/example/* .
 
-* Move `secrets.yaml` out of the git repository. For security reasons,
-  unencrypted secrets should not be committed to
+* Move `secrets.yaml` out of the git repository. For security reasons, unencrypted secrets should not be committed to
   version control.
 
         mv secrets.yaml ..
 
-* Modify `playbook.yaml` and `../secrets.yaml` to customize them to the
-  specifics of your site. The files that you
-  copied contain example configuration values that should be replaced with real
-  email addresses, usernames, *etc.* The
-  files include comments that explain the purpose of each configuration option.
-  If you are not ready to set up your real
-  Data Library server but merely want to practice the installation process,
-  *e.g.* in a virtual machine, you can use the
+* Modify `playbook.yaml` and `../secrets.yaml` to customize them to the specifics of your site. The files you copied 
+  contain example configuration values that should be replaced with real email addresses, usernames, *etc.* The
+  files include comments that explain the purpose of each configuration option. If you are not ready to set up your real
+  Data Library server but merely want to practice the installation process,  *e.g.* in a virtual machine, you can use the
   example files without modification.
 
-* Commit your customizations and push them to your git server for safe keeping;
-  back up `secrets.yaml` by other means,
+* Commit your customizations and push them to your git server for safe keeping; back up `secrets.yaml` by other means, 
```

Comment:
I feel bad when I read this. It essentially says "We don't know how to manage secrets. You're on your own."

Reply:
And introduce yet another password for them to forget, or confuse with their server password and their ssh key encryption password? And if there are multiple admins, how do they share the secrets? I can't recommend git-secret because it's hard to use, but it solves a real problem.